Tools/_sunrise/changelog/changelog_actions.py

```
@@ -18,12 +18,26 @@
 
 
 MAIN_CATEGORY = "Main"
+
+
+def configured_changelog_file() -> Path:
+    value = os.environ.get("CHANGELOG_FILE", "").strip()
+    if not value:
+        raise RuntimeError("Переменная CHANGELOG_FILE не задана")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)